tests/unit/common/test_dynamo_handler.py

```diff
@@ -0,0 +1,35 @@
+import os
+import unittest
+
+from botocore.stub import Stubber
```

Comment:
Why are you using stubber versus https://github.com/spulec/moto for dynamodb?

Reply:
Didn't know about `moto`. Having the ability to verify output in AWS services sounds great - opting for `moto`